nototools/grab_mt_download.py

```diff
@@ -43,7 +43,7 @@
 __author__ = "dougfelt@google.com (Doug Felt)"
 
 import argparse
-import cStringIO
+from io import StringIO
```

Comment:
it's a font, so it is binary data (not unicode text), so you should use `io.BytesIO`

Reply:
That's what I thought and then somehow still put StringIO